File: cpp/funkcje.cpp
/*
 * szablon.cpp
 */


#include <iostream>

using namespace std;

int dodaj(int a, int b)
{
    return a + b;
}
    
int odejmij(int a, int b)
{
    return a - b;
}

int pomnoz(int a, int b)
{
    return a * b;
}

int podziel(int a, int b)
{
    return a / b;
}



int main(int argc, char **argv)
{
	float a, b ;   // deklaracja zmiennej
    a = b = 0;   // inicjacja zmiennej

    cout << "Podaj liczbę 1: ";
    cin >> a;
    cout << a << endl;

    cout << "Podaj liczbę 2: ";
    cin >> b;
    cout << b << endl;

    dodaj(a, b);

    cout << "Suma: " << dodaj(a, b) << endl;
    cout << "Różnica: " << odejmij(a, b) << endl;
    cout << "Iloczyn: " << pomnoz(a, b) << endl;
    cout << "Iloraz: " << podziel(a, b) << endl;
		


	return 0;
}

File: python_inf/funkcje.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
#
#  szablon.py

# zasieg zmiennych moze byc lokalny lub globalny

def dodaj(a, b):
    return a +b
    
def odejmij(a, b):
    return a -b
    
def pomnoz(a, b):
    return a *b
    
def podziel(a, b):
    return a /b

def main(args):
    # a = 0  # inicjacja zmiennej
    a = int(input('Podaj liczbę 1: ')) 
    print(a)
    b = int(input('Podaj liczbę 2: ')) 
    print(b)
    
    
    
    print("Suma: ", dodaj(a, b))
    print("Różnica: ", odejmij(a, b))
    print("Iloczyn: ", pomnoz(a, b))
    print("Iloraz: ", podziel(a, b))
    
    
    return 0
    
    # duck typing

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
